rst_files/wald_friedman.rst

```diff
@@ -510,12 +521,16 @@ Here's the code
     J1 = compute_fixed_point(x -> bellman_operator(pg, 0.5, f0, f1, 5.0, 5.0, x),
         zeros(length(pg)), err_tol=1e-6, print_skip=5);
 
+.. code-block:: julia 
+    :class: test 
 
-Running it produces the following output on our machine
-
-
-
+    @testset "Second Block Tests" begin 
+        @test J1[19] == 0.36
+        @test f1[40] == 0.002163769345396983
```

Comment:
Use `≈` for comparison with `AbstractFloat`

Reply:
Thanks.